docusaurus/docs/introduction/plugin-types-usage.md

```diff
@@ -30,12 +30,13 @@ Grafana plugin development allows for many options depending on the type of user
 - **Data-source plugin** - a connection to a new database or other source of data.
 - **App plugin** - an integrated out-of-the-box experience.
 
+Plugin development is not supported by the Grafana support team. For help, reach out to the [Community forums](https://community.grafana.com/c/plugin-development/30).
```

Comment:
Because I (and I assume most people outside) don't know what the"Grafana support team" this reads to me as "Plugin development is not supported by the grafana team".

Could we change this negative to a positive?:

```For help about plugin development, reach out to the [Community forums](https://community.grafana.com/c/plugin-development/30).```

and perhaps a negative call-to-action? e.g.

```
Don't open support tickets for topics related to plugin development```

Reply:
Okay, but I would put the negative CTA first with the positive CTA to follow (Don't open support tickets for topics related to plugin development. For help with plugin development, reach out to the [Community forums](https://community.grafana.com/c/plugin-development/30).). I've also edited slightly so this is in a Note and a "see also" paragraph is not in a note.